### PR TITLE
Edited flawed comments on all PowerUps (except LifePowerUp).

### PR DESCRIPTION
**Pull request by jorisoudejans**
<documentation>

### CODE REVIEW

**Comment by jorisoudejans** [comment]
By the way, I removed some unused imports, nothing important.

File: src/main/java/doob/model/powerup/FreezeBallsPowerUp.java [hunk 1]
```diff
@@ -33,8 +32,8 @@ public void onDeactivate(Level level) {
     }
 
     /**
-     * Has no duration.
-     * @return 0
+     * The duration of the FreezeBallsPowerUp.
+     * @return DURATION
      */
```
Comment (cbilstra) [inline src/main/java/doob/model/powerup/FreezeBallsPowerUp.java]: Getters and setters do not need javadoc in our project

File: src/main/java/doob/model/powerup/PlayerSpeedPowerUp.java [hunk 2]
```diff
@@ -25,7 +24,7 @@ public void onActivate(Level level, Player player) {
     }
 
     /**
-     * Does nothing.
+     * Sets player speed back to initial value.
      * @param level the level the power-up is in.
```
Comment (cbilstra) [inline src/main/java/doob/model/powerup/PlayerSpeedPowerUp.java]: This javadoc is called something like Return to normal state in the freeze powerup. Maybe caal this one the same?

File: src/main/java/doob/model/powerup/ProjectileFreezePowerUp.java [hunk 3]
```diff
@@ -13,7 +13,7 @@
     public static final int DURATION = 500;
 
     /**
-     * Adds one life to player.
+     * Freezes all projectiles.
      * @param level the level the power-up is in.
```
Comment (cbilstra) [inline src/main/java/doob/model/powerup/ProjectileFreezePowerUp.java]: Only freezes the projectile of one player right?

File: src/main/java/doob/model/powerup/ProjectileSpeedPowerUp.java [hunk 4]
```diff
@@ -1,11 +1,10 @@
 package doob.model.powerup;
 
-import doob.model.Collidable;
 import doob.model.Level;
 import doob.model.Player;
 
 /**
- * Power-up that freezes all balls for a short period of time.
+ * Power-up that speeds up all projectiles for a short period of time.
  */
```
Comment (cbilstra) [inline src/main/java/doob/model/powerup/ProjectileSpeedPowerUp.java]: Only for one player?

File: src/main/java/doob/model/powerup/PlayerSpeedPowerUp.java (hunk 2 above)
Comment (jorisoudejans) [inline src/main/java/doob/model/powerup/PlayerSpeedPowerUp.java]: Not all of them are the same. Some Power-ups have nothing to do on deactivate. But for consistency, I will change all comments on Power-ups that do change onDeactivate to the same syntax.

**Comment by jorisoudejans** [comment]
Have edited it, ready for merge?

**Pull request merged**